developer-manual/i18n.rst

```diff
@@ -19,63 +19,37 @@ repository should respect them.
 
     tx push -s
 
+* Once translators complete their work a new release of
+  :ref:`nethserver-lang-module` can occur.
+
 .. _Transifex: https://www.transifex.com/projects/p/nethserver
 .. _Transifex client: http://docs.transifex.com/developer/client/
 
-To configure the Transifex client execute the `txinit.sh`_ script on the
-repository root.  The script can be executed multiple times, if new
-catalogs are added to the repository.
-
-.. _`txinit.sh`: https://gist.github.com/DavidePrincipi/8e4d4e97831d0850f01a
-
-Currently both gettext and Server Manager specific format is supported
+Currently both gettext and Server Manager specific formats are supported
 for language catalogs.
 
 
-gettext
-=======
+Server Manager (Cockpit)
+========================
 
-The ``xgettext`` command can be used to extract strings from the
-source code. The resulting ``.pot`` file must be named
-:file:`locale/<rpmname>.pot`.
+Packages providing an application for the Cockpit Server Manager must adhere to
+the guidelines documented here:
 
+https://nethserver.github.io/nethserver-cockpit/i18n/#translations
 
-gettext example
----------------
 
-In this example we will translate a new TODO message for the web interface.
-
-Steps to setup translation for a new todo:
-
-1. Be sure gettext is installed in your system
-
-2. Add the todo script to the git repository
+Server Manager (Nethgui)
+========================
 
-3. Create locale directory inside the git repository root:
-
-::
-
-  mkdir locale/
-
-4. Extract the strings (from a Python source code): 
-
-::
-
-  PACKAGE=$(basename `pwd`)
-  xgettext --msgid-bugs-address "translations@nethserver.org" --package-version "1.0.0"  \
-  --package-name "$PACKAGE" --foreign-user -d "$PACKAGE" -o "locale/${PACKAGE}.pot" \
-  -L Python root/etc/nethserver/todos.d/*
-
-5. Run the txinit.sh script and commit newly created files into git.
-
-6. Upload the new ``.pot`` resource to Transifex (special permissions are needed): ::
-
-  tx push -s
+To configure the Transifex client and find the string catalog files
+execute the `txinit.sh`_ script on the repository root.
+The script can run multiple times, when new catalogs are added to the repository.
 
+.. _`txinit.sh`: https://gist.github.com/DavidePrincipi/8e4d4e97831d0850f01a
```

Comment:
Can we move this script inside the nethserver-lang package? I do not like to have it inside a personal gist.

Reply:
I was about to remove completely the "old Server Manager" section because it is no longer developed... Then decided to just reorder the sections and keep them for reference.

But I can change my mind :D

Reply:
The old Server Manager will be alive for 4 more years, so we should keep info on it :)